lib/pipekit/config.rb

```diff
@@ -0,0 +1,41 @@
+module Pipekit
+  class Config
+    class << self
+
+      attr_writer :file_path
+
+      def field(resource, key)
+        custom_fields.fetch(resource, {}).fetch(key.to_s, key)
+      end
+
+      def custom_fields
+        config.fetch("fields", {})
+      end
+
+      def fetch(key)
+        config[key.to_s]
```

Comment:
I was thinking of that but since there's not a specific use case at the moment I thought YAGNI. I can add it in though it would be useful if others use it